make/mshadow.mk

```diff
@@ -25,12 +25,22 @@ ifndef USE_SSE
 	USE_SSE=1
 endif
 
+ifndef USE_F16C
+    USE_F16C=1
```

Comment:
can we detect whether we are compiling for x86 or arm?

Reply:
I could check that yes. 
But I was just thinking, do we need to check if the cpu supports that to check at runtime though? This might be needed if we are cross compiling like we are probably doing for the pip package? Is that needed?

Reply:
Never mind the above, we build different platforms for different platforms, so compile time check should be fine.